packages/main/src/ComboBoxItem.ts

```diff
@@ -52,8 +56,19 @@ class ComboBoxItem extends UI5Element implements IComboBoxItem {
 	@property({ type: Boolean })
 	selected = false;
 
-	get stableDomRef() {
-		return this.getAttribute("stable-dom-ref") || `${this._id}-stable-dom-ref`;
+	/**
+	 * Defines the markup text that will be displayed as suggestion.
+	 * Used for highlighting the matching parts of the text.
+	 *
+	 * @since 2.0.0
+	 * @private
+	 */
+	@property()
+	markupText = "";
+
+	onEnterDOM(): void {
+		super.onEnterDOM();
+		this.setAttribute("accessible-role", "Option");
```

Comment:
This might not work as it is a property of the standard list item only. What you can do is override the listItemAttributes inline template of the ListItemBase.hbs, similarly to the ui5-option 
https://github.com/SAP/ui5-webcomponents/blob/95db5c63a3c21ba2d948f5070f3bd1ba85e584ef/packages/main/src/Option.hbs#L18